mango-cli/src/main/scala/org/bdgenomics/mango/cli/VizReads.scala

```diff
@@ -219,8 +221,17 @@ class VizServlet extends ScalatraServlet {
       if (dictOpt.isDefined && viewRegion.end <= dictOpt.get.length) {
         val sampleIds: List[String] = params("sample").split(",").toList
         val alignments: RDD[AlignmentRecord] = VizReads.readsData.getRaw(viewRegion, sampleIds)
-        val array: Array[AlignmentRecord] = alignments.collect
-        Ok(write(array))
+
+        // convert to GA4GH avro and build a response
+        val gaReads: List[GAReadAlignment] = alignments.map(GA4GHConverter.toGAReadAlignment)
+          .collect
+          .toList
+        val readResponse = GASearchReadsResponse.newBuilder()
+          .setAlignments(gaReads)
+          .build()
+
+        // write response
+        Ok(readResponse)
```

Comment:
What exactly does `write(x)` do? I originally had it as you wrote, but changed it because of the unittest failures I was getting over here.